core/connection_result.h

```diff
@@ -0,0 +1,67 @@
+#ifndef DRONECORE_CONNECTION_RESULT_H
+#define DRONECORE_CONNECTION_RESULT_H
```

Comment:
Usually, I used `#pragma once`.

Reply:
Oh, right. I did not know `#pragma` once actually, and I had seen some guidelines on [isocpp.github.io](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-guards), saying:

> Some implementations offer vendor extensions like `#pragma once` as alternative to include guards. It is not standard and it is not portable.

Do you have an opinion on that? I really don't know, I just read it. I can change it for a `pragma` if you prefer.

Reply:
I prefer it because it seems to work and it prevents errors when you copy and paste files without changing the define correctly.

Reply:
ok